### PR TITLE
fix(curriculum): add word nesting & comments to nutrition-label-step-43

### LINKED ISSUE #49017: Nutrition Label Step 43
### Describe the Issue

The forum is getting hammered with questions for Nutrition Label Step 43. I think we can improve the last sentence to help with this.

### Affected Page

https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-typography-by-building-a-nutrition-label/step-43

### Current Instructions

This is the current instructions:

> Step 43

> After your last `.divider` element, create a `p` element and give it the text `Total Fat 8g 10%`. Wrap `Total Fat` in a `span` element with the `class` set to `bold`. Wrap `10%` in another `span` element with the `class` set to `bold`. Also wrap the words `Total Fat 8g` in an additional `span` element for alignment.

### Expected behavior

Users are not sure where to put the third span element.

I think we can use the phrase `nesting` in the last sentence to make the intended solution clear:

> Finally, nest these two `span` elements in an additional `span` element for alignment.

If this doesn't work, we may want to consider splitting this step into two parts.

### Screenshots

_No response_

### System

All

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by marktlinn**
Add comments to code to clear up confusion about where user's should actually be writing their code, as this has been the cause of problems and frustration for some. Also improve the description to incorporate the use of the phrasing 'nesting' in a clear and understandable manner as per suggestions in issue #49017

Checklist:

- [ X ] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [ X ] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [ X ] My pull request targets the `main` branch of freeCodeCamp.
- [  X ] I have tested these changes either locally on my machine, or GitPod.

fix #49017

Improve the description to incorporate the use of the phrasing 'nesting' in a clear and understandable manner as per suggestions in issue #49017. Also, added comments to code to clear up confusion about where user's should actually be writing their code, as this has been the cause of problems and frustration for some.

### CODE REVIEW

File: curriculum/challenges/english/14-responsive-web-design-22/learn-typography-by-building-a-nutrition-label/615f6b2d164f81809efd9bdc.md [hunk 1]
```diff
@@ -95,6 +95,9 @@ assert(document.querySelector('.daily-value.small-text')?.lastElementChild?.inne
     <div class="daily-value small-text">
       <p class="bold right">% Daily Value *</p>
       <div class="divider"></div>
+      <!--Add your code below this line-->
+      
+      <!--Add your code above this line-->
```
Comment (Sboonny) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-typography-by-building-a-nutrition-label/615f6b2d164f81809efd9bdc.md:100]:
```suggestion
```

File: curriculum/challenges/english/14-responsive-web-design-22/learn-typography-by-building-a-nutrition-label/615f6b2d164f81809efd9bdc.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ dashedName: step-43
 
 # --description--
 
-After your last `.divider` element, create a `p` element and give it the text `Total Fat 8g 10%`. Wrap `Total Fat` in a `span` element with the `class` set to `bold`. Wrap `10%` in another `span` element with the `class` set to `bold`. Also wrap the words `Total Fat 8g` in an additional `span` element for alignment.
+After your last `.divider` element, create a `p` element and give it the text `Total Fat 8g 10%`. Wrap `Total Fat` in a `span` element with the `class` set to `bold`. Wrap `10%` in another `span` element with the `class` set to `bold`. Finally wrap both the `span` element `Total Fat` and the text `8g` with a `span`, so that they are nested together inside a `span` element.
```
Comment (Sboonny) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-typography-by-building-a-nutrition-label/615f6b2d164f81809efd9bdc.md:10]:
```suggestion
After your last `.divider` element, create a `p` element and give it the text `Total Fat 8g 10%`. Wrap `Total Fat` in a `span` element with the `class` set to `bold`. Wrap `10%` in another `span` element with the `class` set to `bold`. Finally, nest these two `span` elements in an additional `span` element for alignment.
```

**Review by Sboonny — CHANGES_REQUESTED** [review]
To solve this issue:
- change the wording to what agreed upon in the issue.
- change step 53 as well.

---

Can you edit the PR title to reflect on our conviction, this isn't how you add scope

**Comment by marksist300** [comment]
> To solve this issue:
> 
>     * change the wording to what agreed upon in the issue.
> 
>     * change step 53 as well.
> 
> 
> Can you edit the PR title to reflect on our conviction, this isn't how you add scope

Yes, of course and thanks for the comment. I've made the changes, I hope all is ok now. Let me know if there's anything else I can change or improve 🙂. I'll take a look at step now 53 too.

**Comment by marksist300** [comment]
> To solve this issue:
> 
>     * change the wording to what agreed upon in the issue.
> 
>     * change step 53 as well.
> 
> 
> Can you edit the PR title to reflect on our conviction, this isn't how you add scope

@Sboonny Done, added a similar description to step 53 👍🏻

**Review by Sboonny — APPROVED** [review]
LGTM 🎉, thank you for your contribution to the page! 👍.

File: curriculum/challenges/english/14-responsive-web-design-22/learn-typography-by-building-a-nutrition-label/615f6b2d164f81809efd9bdc.md [hunk 3]
```diff
@@ -7,7 +7,7 @@ dashedName: step-43
 
 # --description--
 
-After your last `.divider` element, create a `p` element and give it the text `Total Fat 8g 10%`. Wrap `Total Fat` in a `span` element with the `class` set to `bold`. Wrap `10%` in another `span` element with the `class` set to `bold`. Also wrap the words `Total Fat 8g` in an additional `span` element for alignment.
+After your last `.divider` element, create a `p` element and give it the text `Total Fat 8g 10%`. Wrap `Total Fat` in a `span` element with the `class` set to `bold`. Wrap `10%` in another `span` element with the `class` set to `bold`. Finally, nest these two `span` elements in an additional `span` element for alignment.
```
Comment (ShaunSHamilton) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-typography-by-building-a-nutrition-label/615f6b2d164f81809efd9bdc.md:10]: The desired outcome is:

```html
<p>
  <span>
    <span class="bold">Total Fat</span> 8g
  </span> 
  <span class="bold">10%</span>
</p>
```

These instructions suggest:

```html
<p>
  <span>
    <span class="bold">Total Fat</span> 8g <span class="bold">10%</span>
  </span>
</p>
```

**Review by ShaunSHamilton — CHANGES_REQUESTED** [review]
Step 43 needs more clarification.

File: curriculum/challenges/english/14-responsive-web-design-22/learn-typography-by-building-a-nutrition-label/615f6b2d164f81809efd9bdc.md (hunk 3 above)
Comment (marksist300) [reply curriculum/challenges/english/14-responsive-web-design-22/learn-typography-by-building-a-nutrition-label/615f6b2d164f81809efd9bdc.md]: @ShaunSHamilton Yes, you're right! I have tried to rectify it. Hopefully the wording is more accurate and helpful now. Let me know if there's anything I can do to improve it. Thanks for the feedback! 🙂

**Review by ShaunSHamilton — APPROVED** [review]
This LGTM 👍 

Thank you for making that change, and for this contribution 🙂

**Pull request merged**